jep/200/README.adoc

```diff
@@ -123,6 +123,18 @@ When a class is (first) rejected, a warning is printed to the system log.
 This is particularly useful when scanning large automated test results for regressions caused by this change.
 Typically the rejection will also trigger an exception which gets displayed in some other manner.
 
+== Extensibility
+
+Plugin developers will be able to whitelist the classes in their libraries/plugins using
+the following approaches:
+
+0. `Jenkins-ClassFilter-Whitelisted` manifest entry (link:https://github.com/jenkinsci/lib-jenkins-maven-embedder/pull/15[example for Maven])
+0. By providing a custom `META-INF/hudson.remoting.ClassFilter` resource file in the plugin
+0. By implementing the `CustomClassFilter` extension point and implementing `permits()` methods
```

Comment:
Not _currently_ an option as it is `@Restricted`, pending a concrete use case.

Reply:
"Permissive-ClassFilter-Plugin" similar to https://wiki.jenkins.io/display/JENKINS/Permissive+Script+Security+Plugin? :) Maybe helpful for extra logging.

Or just a custom management of whitelisting for classes similar to the resource file

Reply:
As I said, there is no concrete use case. If and when one is written, the API can be exposed.

Reply:
Maybe you want to comment on the use-case I proposed above. But I am fine with adjusting the section to the current state in order to indicate that the API is restricted

Reply:
My comment is: please no. This is not something a Jenkins user should be doing.